storybookjs/storybook
# Docs: Minor updates to the Environment variables docs

With this pull request, the Environment variables documentation was updated to factor in some information about Vite and how to obtain environment variables specific to this builder.

What was done:
- Minor wording tweaks
- Consolidated the snippets to become framework agnostic to reduce the maintenance cost
-  Introduced conditional rendering of the documentation to avoid confusion

Closes #23528

@IanVS, when you have a moment, can you take a look and let me know of any feedback you have? Appreciate it

## How to test

1. Follow the steps in the [contributing instructions](https://storybook.js.org/docs/react/contribute/new-snippets#preview-your-work) for this branch, `chore_env_vars_tweaks`.


2. Open the relevant documentation, and for Vite frameworks, you should see a similar section added to the documentation

![framework-conditional-render-vite-support](https://github.com/storybookjs/storybook/assets/22988955/1c4a5a09-554c-4895-9797-788e968ccc16)

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers


`["cleanup", "BREAKING CHANGE", "feature request", "bug", "build", "documentation", "maintenance", "dependencies", "other"]`



### 🦋 Canary release



This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_
